assin.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+<?php
+$f_name =  $_POST["fname"];       //store first name via post method
+$l_name =  $_POST["lname"];       //store last name via post method
+$full_name = $f_name." ".$l_name; //concatenate the fname and lname
```

Comment:
Why do we need to concat first name and last name if we have a field for full name already in the form?